cl_sii/rut/crypto_utils.py

```diff
@@ -0,0 +1,47 @@
+import re
+from typing import Optional
+
+import cryptography
+import cryptography.x509
+from cryptography.hazmat.backends.openssl import backend as crypto_x509_backend
+
+from . import constants
+from . import Rut
+
+
+def get_subject_rut_from_certificate_pfx(pfx_file_bytes: bytes, password: Optional[str]) -> Rut:
```

Comment:
We should add a link to the original source either in the function's docstring or in the commit message.

Original source URL: https://github.com/fyntex/fd-cl-data/blob/cfd5a716fb9b2cbd8a03fca1bacfd1b844b1337f/fd_cl_data/apps/sii_auth/models/sii_auth_credential.py#L701-L745

Reply:
You're absolutely right, I'm usually hesitant to add references to private repositories in this repository, but the fair thing to do is to always reference the source of what we use.

Reply:
> [...], I'm usually hesitant to add references to private repositories in this repository, [...]

You're right about that. In the future, we could hide the internal design of the original source's project by replacing the URL pointing to a file with one pointing to a commit: https://github.com/fyntex/fd-cl-data/commit/9bf4f0fb#diff-113c7101778ef51e6d244611ed664881aa6260b4ab7be54869d0c9701bb6706bR329-R373